cms/search/publishers.py

```diff
@@ -97,12 +93,17 @@ def _construct_message_for_create_update(self, page: "Page") -> dict:
     def _construct_release_specific_fields(page: "Page") -> dict:
         """Constructs and returns a dictionary with release-specific fields."""
         release_fields = {
+            "provisional_date": None,
```

Comment:
This is a string in the api; however, let's omit this. Only add if needed as done below.

Reply:
Done!